sympy/core/evalf.py

```diff
@@ -77,7 +78,7 @@ class PrecisionExhausted(ArithmeticError):
 represent the mantissa (man) in binary notation, e.g.
 """
 
-MPF_TUP = tuple[int, int, int, int]  # mpf value tuple
+MPF_TUP: TypeAlias = tuple[int, int, int, int]  # mpf value tuple
```

Comment:
Does it help in some way to have TypeAlias here?

Reply:
One would think making type alias unambiguous should be a good thing at all times, but

https://github.com/sympy/sympy/pull/28267#discussion_r2261552321
> [...] I didn't pick and choose which ones to add, I just merged what had been identified from https://github.com/microsoft/python-type-stubs/tree/main/stubs/sympy-stubs .

> 
> Between [the] odd Sphinx warning, and needing to import behind TYPE_CHECKING. Maybe it's best to omit until 3.10 is the minimum supported Python version, or there's a specific need to disambiguate from a value alias/assignement. Type-checkers already "try their best" to differentiate.

So I'll just remove the added TypeAlias from this PR either way.